### PR TITLE
Add yaml file to .github/workflows/ci.yml

### PR DESCRIPTION
**Pull request by sebastianpiresmolin**
Added a yaml folder containing ci.yml to set up github actions.

image

### CODE REVIEW

**Comment by rasmusbaeckstrom** [comment]
Exciting! Cant see anything strange here. Good job.

File: .github/workflows/ci.ym [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```
Comment (richardolsson) [inline .github/workflows/ci.ym:19]: You are specifying to use use Node version 14 here, but that's a quite old version by now. Node 14 was released in 2020 and stopped receiving updates in 2023. The current version of Node with long-term support (LTS) is v20. I'm guessing you've found an old tutorial and used the values from that, which is an easy mistake to make.

File: .github/workflows/ci.ym [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (richardolsson) [inline .github/workflows/ci.ym:6]: I would expect this to trigger the CI for any PR that targets `main`, but it looks to me like this PR didn't trigger the CI, and that it kept not working until the changes in #6.

**Review by richardolsson — COMMENTED** [review]
Nice work on this and your subsequent changes to it in other PRs. I've added some comments below.

File: .github/workflows/ci.ym (hunk 2 above)
Comment (sebastianpiresmolin) [reply .github/workflows/ci.ym:6]: Yes, at first I had named my yaml file ".ym" instead of ".yml" and the tutorial i followed didn't behave the way I intended it to, so I made many small commits which makes it hard to follow the development flow.

If you have any tips on how to test something like this without having to actually push and merging it first I would appreciate to hear about it.

Comment (richardolsson) [reply .github/workflows/ci.ym:6]: Wow I didn't even realize the file name was incorrect. Thanks for pointing that out.

I'm afraid there is no other way to reliably test GitHub actions than to push to a PR, but I think that's fine. Any work you do in that PR will have immediate effect on the actions for that PR, and once you're happy you can squash and merge so that it looks in the Git history as if you only made a single commit.
